benchmark.go

```diff
@@ -39,8 +40,37 @@ import (
 var (
 	errNegativeDuration = errors.New("duration cannot be negative")
 	errNegativeMaxReqs  = errors.New("max requests cannot be negative")
+
+	// using a global quantiles array mainly for ease of testing, and not passing the same array
+	// around to multiple functions
+	// TODO: modify quantile parameters to remove "." character and increase ease of parsing
+	_quantiles = []float64{0.5000, 0.9000, 0.9500, 0.9900, 0.9990, 0.9995, 1.0000}
```

Comment:
this slice can just be the old `[]float64{0.5, 0.9, 0.95, 0.99, 0.999, 0.9995, 1.0}` right?